timesketch/lib/datastores/opensearch.py

```diff
@@ -705,8 +728,13 @@ def search(
                 cause = ", ".join(error_items)
             else:
                 cause = str(e)
-
-            es_logger.error("Unable to run search query: %s", cause, exc_info=True)
+            log_message = (
+                f"Unable to run search query. Error: {cause}. "
+                f"Sketch ID: {sketch_id}. Indices: {indices}. "
+                f"Query String: '{query_string}'. Query Filter: {query_filter}. "
+                f"Query DSL: {query_dsl}"
+            )
+            os_logger.error(log_message, exc_info=True)
             raise ValueError(cause) from e
```

Comment:
Do we also want to improve the error message that is presented to the user in the red snackbar? This would need to go into the `ValueError` here.

Reply:
I was thinking about that as well, happy to change and yeah I think that would help

Reply:
yes, please add at least an info that something in their query was not legit and the user should revisit their search query.

Reply:
done